lib/beacon/docker-compose.beacon.yaml

```diff
@@ -24,12 +23,15 @@ services:
       - BENTOV2_DOMAIN
       - BENTOV2_PUBLIC_URL
       - BENTO_BEACON_UI_ENABLED
+      - BENTO_AUTHZ_SERVICE_URL
+      - DRS_URL=${BENTOV2_PUBLIC_URL}/api/drs
     volumes:
       - ${BENTO_BEACON_CONFIG_DIR}:/config:ro
     networks: 
       - beacon-net
       - gohan-api-net
       - katsu-net
+      - drs-net
```

Comment:
DRS network shouldn't be needed if it goes through the gateway, which it looks like it does looking at DRS_URL

Reply:
ah, indeed not, I had the network change first